server/prisma/generator/factories/instanceRoles.factory.ts

```diff
@@ -36,18 +49,20 @@ const createRole = async ({ name, permissions }: InstanceRole) => {
       instance_role_permissions: { create: permissionsData },
     },
   });
-  return { [name]: { name: createdRole.name, id: createdRole.id } };
+  const role: RoleMap = {};
+  role[name] = createdRole;
+  return role as RoleMap;
```

Comment:
I'm curious about this part. Is there some specific reason why `role` is both declared as `RoleMap` and then returned `as RoleMap`?

Reply:
Uh, not a great reason. I think I was just trying a few things to get the types to play nicely and forgot about this. I'll try dropping it since it seems redundant.